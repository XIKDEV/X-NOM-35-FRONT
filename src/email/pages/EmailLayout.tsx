import '../ui/emailLayout.css';

export const EmailLayout = () => {
  return (
    <div className="email-layout flex-column-center">
      <img src="../../../public/XIK_VerdeTransparente.png" alt="XikDev" />
      <h1>XikDev</h1>

      <div className="email-text flex-column-center">
        <p>
          Bienvenido! <br />
          Por medio de este correo se comparte de manera automática la
          contraseña correspondiente para entrar al sistema de Nom-035. <br />
          Ingresa a [anexar link] o da click al siguiente botón para dirigirte a
          la página e iniciar sesión.
        </p>

        <div className="password flex-column-center">
          <p>Contraseña:</p>
          <p>AKJC64cdsK5!</p>
        </div>

        <a>Nom-035</a>

        <p>
          <span>
            Este mensaje es generado de manera automática, favor de no responder
            directamente este correo, para realizar alguna consulta o tratar
            algún tema, enviar correo a xikdev@gmail.com
          </span>
        </p>
      </div>
    </div>
  );
};
