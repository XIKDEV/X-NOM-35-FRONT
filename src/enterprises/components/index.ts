export * from './ContractsList';
export * from './EnterprisesCard';
export * from './HeadEnterpriseInfo';
export * from './InfoEnterprise';
