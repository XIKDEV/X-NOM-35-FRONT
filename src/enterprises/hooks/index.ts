export * from './useEnterprisesPage';
export * from './useFormEnterprise';
