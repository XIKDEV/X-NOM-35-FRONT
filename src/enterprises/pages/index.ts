export * from './EnterpriseInfoPage';
export * from './EnterprisesPage';
