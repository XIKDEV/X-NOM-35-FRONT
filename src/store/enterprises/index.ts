export * from './enterprisesSlice';
export * from './thunks';
