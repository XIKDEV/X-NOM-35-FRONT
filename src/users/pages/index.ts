export * from './UserInfoPage';
export * from './UsersPage';
