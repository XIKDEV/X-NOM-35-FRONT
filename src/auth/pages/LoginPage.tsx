import { LoginPageHooks } from '../hooks';

import { authConstants } from '../../constants/authConstants';

import '../ui/loginPage.css';

export const LoginPage = () => {
  LoginPageHooks();

  return (
    <div className="login">
      <form>
        <img
          className="logo"
          src="../../../public/XIK_VerdeTransparente.png"
          alt="XikDev"
        />
        <div className="login-inputs">
          <input type="email" placeholder="Email" />
          <input type="password" placeholder="ContraseÃ±a" />
        </div>
        <button>{authConstants.bttnLogin}</button>
      </form>
    </div>
  );
};
