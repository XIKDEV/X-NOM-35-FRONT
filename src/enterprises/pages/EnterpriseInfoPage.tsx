import { FC } from 'react';
import { Layout } from 'antd';
import { HeaderMobile } from '../../ui/components';
import {
  ContractsList,
  HeadEnterpriseInfo,
  InfoEnterprise,
} from '../components';
import { useScreenSize } from '../../hooks';
import { IEnterpriseInfoPageProps } from '../../interfaces';
import { enterpriseContentStyle } from '../ui';
import '../ui/enterpriseInfoPage.css';

const { Content } = Layout;

export const EnterpriseInfoPage: FC<IEnterpriseInfoPageProps> = ({
  handleSidebar,
}) => {
  const { isMobile } = useScreenSize();

  // //! TODO: temp info
  const enterpriseInfo = {
    enterpriseName: 'XikDev',
    addres: 'Agustín Lara #3251 Col. Miramar, Tijuana, Baja California, México',
    goals: 'Goals',
    activity: 'Industria Tec',
    mainActivity: 'Desarrollo de web',
    quantityEmployees: '2',
    contractsData: [
      {
        contract: 'Nom-contract',
        date: 'mm/aa - mm/aa',
      },
      {
        contract: 'Nom-contract',
        date: 'mm/aa - mm/aa',
      },
      {
        contract: 'Nom-contract',
        date: 'mm/aa - mm/aa',
      },
    ],
  };

  return (
    <>
      {isMobile ? <HeaderMobile handleSidebar={handleSidebar} /> : <></>}
      <HeadEnterpriseInfo enterpriseName={enterpriseInfo.enterpriseName} />
      {isMobile ? (
        <>
          <InfoEnterprise
            addres={enterpriseInfo.addres}
            goals={enterpriseInfo.goals}
            activity={enterpriseInfo.activity}
            mainActivity={enterpriseInfo.mainActivity}
            quantityEmployees={enterpriseInfo.quantityEmployees}
          />
          <ContractsList data={enterpriseInfo.contractsData} />
        </>
      ) : (
        <Content style={enterpriseContentStyle}>
          <InfoEnterprise
            addres={enterpriseInfo.addres}
            goals={enterpriseInfo.goals}
            activity={enterpriseInfo.activity}
            mainActivity={enterpriseInfo.mainActivity}
            quantityEmployees={enterpriseInfo.quantityEmployees}
          />
          <ContractsList data={enterpriseInfo.contractsData} />
        </Content>
      )}
    </>
  );
};
