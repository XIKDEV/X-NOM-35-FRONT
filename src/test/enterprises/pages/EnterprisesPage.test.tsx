import { render } from '@testing-library/react';
import { EnterprisesPage } from '../../../enterprises/pages';

describe('Test in <EnterprisesPage />', () => {
  test('should be match with snapshot', () => {
    const { asFragment } = render(<EnterprisesPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
