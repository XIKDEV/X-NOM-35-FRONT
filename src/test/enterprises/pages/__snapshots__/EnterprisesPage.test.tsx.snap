// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <EnterprisesPage /> should be match with the snapshot 1`] = `
<DocumentFragment>
  <header
    class="ant-layout-header header-desktop flex-row-center justify-end css-dev-only-do-not-override-17a39f8"
  >
    <span
      class="ant-input-affix-wrapper header-desktop-input br-8 ant-input-affix-wrapper-borderless css-dev-only-do-not-override-17a39f8"
    >
      <input
        class="ant-input ant-input-borderless css-dev-only-do-not-override-17a39f8"
        placeholder="Búsqueda rápida"
        type="text"
        value=""
      />
      <span
        class="ant-input-suffix"
      >
        <span
          class="ant-input-clear-icon ant-input-clear-icon-hidden"
          role="button"
          tabindex="-1"
        >
          <span
            aria-label="close-circle"
            class="anticon anticon-close-circle"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="close-circle"
              fill="currentColor"
              fill-rule="evenodd"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64c247.4 0 448 200.6 448 448S759.4 960 512 960 64 759.4 64 512 264.6 64 512 64zm127.98 274.82h-.04l-.08.06L512 466.75 384.14 338.88c-.04-.05-.06-.06-.08-.06a.12.12 0 00-.07 0c-.03 0-.05.01-.09.05l-45.02 45.02a.2.2 0 00-.05.09.12.12 0 000 .07v.02a.27.27 0 00.06.06L466.75 512 338.88 639.86c-.05.04-.06.06-.06.08a.12.12 0 000 .07c0 .03.01.05.05.09l45.02 45.02a.2.2 0 00.09.05.12.12 0 00.07 0c.02 0 .04-.01.08-.05L512 557.25l127.86 127.87c.04.04.06.05.08.05a.12.12 0 00.07 0c.03 0 .05-.01.09-.05l45.02-45.02a.2.2 0 00.05-.09.12.12 0 000-.07v-.02a.27.27 0 00-.05-.06L557.25 512l127.87-127.86c.04-.04.05-.06.05-.08a.12.12 0 000-.07c0-.03-.01-.05-.05-.09l-45.02-45.02a.2.2 0 00-.09-.05.12.12 0 00-.07 0z"
              />
            </svg>
          </span>
        </span>
      </span>
    </span>
    <button
      class="br-8"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-plus "
        data-icon="plus"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
          fill="currentColor"
        />
      </svg>
    </button>
  </header>
  <section
    class="grid-enterprises-cards"
  >
    <div
      class="ant-card ant-card-bordered ant-card-hoverable card-enterprise br-8 css-dev-only-do-not-override-17a39f8"
    >
      <div
        class="ant-card-cover"
      >
        <img
          alt="Logo de XikDev"
          src="../../../public/_DSC7606.JPG"
        />
      </div>
      <div
        class="ant-card-body"
        style="padding: 12px;"
      >
        <div>
          MockedMetaComponent
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;

exports[`Test in <EnterprisesPage /> should render <HeaderMobile /> 1`] = `
<div>
  <header
    class="ant-layout-header header-mobile flex-row-center space-between css-dev-only-do-not-override-17a39f8"
  >
    <button
      data-testid="sidebar-toggle-button"
    >
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-bars "
        data-icon="bars"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 96C0 78.3 14.3 64 32 64H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H32C14.3 128 0 113.7 0 96zM0 256c0-17.7 14.3-32 32-32H416c17.7 0 32 14.3 32 32s-14.3 32-32 32H32c-17.7 0-32-14.3-32-32zM448 416c0 17.7-14.3 32-32 32H32c-17.7 0-32-14.3-32-32s14.3-32 32-32H416c17.7 0 32 14.3 32 32z"
          fill="currentColor"
        />
      </svg>
    </button>
    <input
      class="br-4"
      placeholder="Búsqueda rápida"
      type="text"
    />
    <button>
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-plus "
        data-icon="plus"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
          fill="currentColor"
        />
      </svg>
    </button>
  </header>
  <div
    class="ant-card ant-card-bordered card-mobile br-4 css-dev-only-do-not-override-17a39f8"
  >
    <div
      class="ant-card-body"
      style="padding: 0px; height: 100%; display: flex; align-items: center;"
    >
      <img
        alt="Logo de XikDev"
        class="br-4"
        src="../../../public/_DSC7606.JPG"
      />
      <div>
        MockedMetaComponent
      </div>
    </div>
  </div>
</div>
`;
